ssh/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 9.2.1
+
+- Upgrade Home Assistant CLI to 4.14.0
```

Comment:
This includes the datadisk commands which require a supervisor version that is currently not on stable

Reply:
I can't suggest for some reason :thinking: 
Should we make the version a link?
`[4.14.0](https://github.com/home-assistant/cli/releases/tag/4.14.0)`

Reply:
ah, it's merged already :+1:

Reply:
right, but also not will break anything